include/votca/xtp/fileadapter.h

```diff
@@ -0,0 +1,211 @@
+/*                                                                              
+ *            Copyright 2009-2019 The VOTCA Development Team                    
+ *                       (http://www.votca.org)                                 
+ *                                                                              
+ *      Licensed under the Apache License, Version 2.0 (the "License")          
+ *                                                                              
+ * You may not use this file except in compliance with the License.             
+ * You may obtain a copy of the License at                                      
+ *                                                                              
+ *              http://www.apache.org/licenses/LICENSE-2.0                      
+ *                                                                              
+ * Unless required by applicable law or agreed to in writing, software          
+ * distributed under the License is distributed on an "AS IS" BASIS,            
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.        
+ * See the License for the specific language governing permissions and          
+ * limitations under the License.                                               
+ *                                                                              
+ */  
+
+#pragma once
+#ifndef VOTCA_XTP_FILEADAPTER_H
+#define VOTCA_XTP_FILEADAPTER_H
+
+#include <algorithm>
+#include <cassert>
+#include <string>
+#include <vector>
+#include <unordered_map>
+#include <votca/tools/unitconverter.h>
+#include <votca/tools/structureparameters.h>
+
+#include <Eigen/Dense>
+
+namespace votca {
+  namespace xtp {
+
+
+    /**
+     * @brief File Topology Adapter
+     *
+     * This class is designed to interface with the topology structures in xtp
+     * and in csg such that the xtp classes can use the csg topolology and 
+     * trajectory input and output readers and writers
+     */
+    class FileAdapter {
```

Comment:
This is a topology converter not a FileAdapter

Reply:
True enough will fix names.